test/form/samples/system-multiple-export-bindings/_expected.js

```diff
@@ -0,0 +1,54 @@
+System.register([], function (exports) {
+	'use strict';
+	return {
+		execute: function () {
+
+			exports({
+				fn: fn,
+				fn2: fn
+			});
+
+			const x = exports('x', 123);
+			const y = exports('y', 456);
+
+			var namespace = /*#__PURE__*/Object.freeze({
+				__proto__: null,
+				x: x,
+				y: y
+			});
+			exports({namespace: namespace, namespace2: namespace});
+
+			// Namespace variable
+
+			// Variable Declaration
+			let a = function (v) {exports({a: a, a2: a}); return v;} ( 1), b = exports('b', 2), c = function (v) {exports({c: c, c2: c}); return v;} ( 3);
+
+			// Export default expression
+			var a$1 = exports('default', a);
+
+			// Assignment Expression
+			a = function (v) {exports({a: a, a2: a}); return v;} ( b = exports('b', c = function (v) {exports({c: c, c2: c}); return v;} ( 0)));
```

Comment:
@guybedford  Why is default not updated here? Is that default expression handled differently than named bindings?

Reply:
Default export expressions are snapshots of an expression value at the point of declaration